samples/jwt/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
This comment is not related to this file, but in general to the sample.

Maybe the name of the sample could also be `app_jwt`, to avoid confusion with modem JWT.

I was also wondering about the location of the sample. I noticed that there's `ssf_client` sample in the `sdfw` directory. This sample uses the same SDFW service framework as this one. So, should this sample be under the same directory? And can this also be made to work with nRF54H20?